src/app/my-info/(account)/reviews/_components/MyReviewProduct.tsx

```diff
@@ -1,52 +1,131 @@
 'use client';
 
+import { deleteUserProductReview } from '@/api/productReviewAPI';
+import { Dialog, Modal } from '@/components';
+import WriteEditModal from '@/components/WriteEditModal/WriteEditModal';
+
 import { formatDateToString } from '@/libs/formatDateToString';
-import { ProductType } from '@/types/ProductTypes';
+
+import type { ReviewDto } from '@/types/ProductReviewTypes';
+
+import { QueryClient, useMutation } from '@tanstack/react-query';
 import classNames from 'classnames/bind';
 import Image from 'next/image';
 import Link from 'next/link';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { ProductType } from '@/types/ProductTypes';
 import styles from './MyReviewProduct.module.scss';
 
 const cn = classNames.bind(styles);
 
 interface MyReviewProductProps {
-  productId: number;
-  updatedAt: Date;
-  switchOption: string;
+  reviewData: ReviewDto;
   productData?: ProductType;
 }
 
-export default function MyReviewProduct({ productId, updatedAt, switchOption, productData }: MyReviewProductProps) {
-  if (!productData) {
-    return <div className={cn('no-product-data')}>상품 정보를 불러올 수 없습니다.</div>;
-  }
+const defaultProductData = {
+  name: '상품 정보 없음',
+  thubmnailList: [{ id: 0, imgUrl: '/public/images/kedeukProfile.png' }],
+  categoryName: 'keyboard',
+};
+
+export default function MyReviewProduct({ reviewData, productData }: MyReviewProductProps) {
+  const { name: productName, thubmnailList, categoryName } = productData || defaultProductData;
```

Comment:
```suggestion
  const { name: productName, thubmnailList, categoryName } = productData ?? defaultProductData;
```
falsy값을 판단하는 ||연산자 보다는 해당 구문에서는 null과 undefined만 판단하는 ??연산자 사용이 더 나을거같아요